cli/command/service_test.go

```diff
@@ -138,34 +110,21 @@ func TestReadService(t *testing.T) {
 		ReadService("svc_id").
 		Return(&models.Service{}, nil)
 
-	c := testutils.NewTestContext(t, []string{"svc_name"}, nil)
-	if err := command.read(c); err != nil {
+	if err := testutils.RunApp(command, "l0 service get svc_name"); err != nil {
 		t.Fatal(err)
 	}
 }
 
 func TestReadServiceInputErrors(t *testing.T) {
```

Comment:
Should we change get to read? for the cli?

Reply:
I was thinking about it, but I figured why not just take care of that whenever we do https://github.com/quintilesims/layer0/issues/505